tools/distro-json-generate.mjs

```diff
@@ -46,28 +75,36 @@ async function updateJsonFile(file, newData) {
 
   try {
     oldData = fs.existsSync(file) ? await fs.readFile(file, 'utf8') : null;
-    // Eliminate formatting
-    oldData = oldData?.replace(/\s/g, '') ?? null;
+    // Eliminate formatting. removes WS in the beginning, end. before & after non characters
+    oldData = oldData?.replace(/^\s|\s$|\B\s|\s\B/g, '') ?? null;
```

Comment:
why do we need this here? can we have those not added?

Reply:
used only to remove formatting before comparing old and new data and prevent writing to disk the same data every run

Reply:
dropped it all together
as per:
> I think we don't need this here, should be covered by normal unit tests.